record/record.app.js

```diff
@@ -58,6 +60,53 @@
 
         ERMrest.appLinkFn(UriUtils.appTagToURL);
 
+        function getPageSize(obj){
+            return ((!angular.isUndefined(obj) && obj.display.defaultPageSize)?obj.display.defaultPageSize:constants.defaultPageSize);
+        }
+
+        function getRelatedTableData(refObj,displaytype,accordionOpen,callback){
```

Comment:
Isn't display type something that can be derived from the reference object? I don't think you need to pass the "displaytype". Also add documentation for this function.